data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,436 @@
+				accept
+	on stopover
+		conversation
+			`You have finally visited every world in Hai space and sold the pitch to the people. With any luck enough people will sign up to make it work.`
+			`	Terry indicates that you may now take her to Mountaintop without further stops, but she has some leftover materials so she'll get you to do some final spruiking if you stop anywhere else along the way.`
+	on complete
+		log `You had to go on a lengthy promotional tour to rustle up volunteers for the new station. Terry took care of all the details and seemed especially keen. Presumably she has some marketing background, which may explain her success within the Syndicate.`
+		payment 10000000
+		conversation
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop. Shortly after you receive a ping informing you that <payment> has been deposited into your account. Terry must've set it up to happen automatically.`
+				decline
+
+
+
+mission "ZZ Hai Building Mountaintop [2a] Sales Job"
+	landing
+	invisible
+	repeat
+	source 
+		not planet "Hai-home"
+		not planet "Mountaintop"
+		not planet "Darkwaste"
+		government "Hai"
+	to offer
+		has "Hai Building Mountaintop [2] Recruits: active"
+	on offer
+		conversation
+			`You land on the world of <planet> and do your part to sell the opportunity on Mountaintop.`
+				decline
+
+
+
+mission "Hai Building Mountaintop [3] Furnishings"
+	name "Retrieve human furnishings."
+	description "The volunteers are so numerous that the Hai need outside help transporting furnishings for their quarters to the new station."
+	autosave
+	landing
+	source "Mountaintop"
+	stopover "Prime"
+	destination "Mountaintop"
+	clearance
+	on visit
+		dialog `You have arrived at the Mountaintop construction site but it isn't finished or staffed yet.`
+	to offer
+		has "Hai Building Mountaintop [2] Recruits: done"
+	cargo "human furnishings" 200
+	blocked `Sayari contacts you and says, "<first>, you're going to need two hundred tons of cargo space for this." You should return here after freeing up <capacity>.`
+	on offer
+		conversation
+			`As you and Terry offload her remaining supplies on Mountaintop where she intends to manage the arrivals and setup personally, you get a communication from Sayari come to your personal device.`
+			`	"Captain <last>! Glad I could get a hold of you on the other side of the wormhole there, I see the new relays are working."`
+			`	"We've just discovered a problem, interest has soared in sign-ups for Mountaintop, but we've realized that most of our 'human' furnishings produced in Hai space aren't going to pass as having been manufactured with your technology. Mr Yang has pulled some strings, but now we urgently need someone to go pick them up from Prime in Betelgeuse."`
+			`	"Everything else should be ready on our end soon enough, if you can just handle that pick-up and delivery we'll be ok."`
+			`	You assure her you can handle a simple pick-up.`
```

Comment:
```suggestion
			`	You let her know you'll handle it and prepare to leave.`
```
To me, the current wording makes it sound like `<first>` has to convince Sayari they can handle it, but Sayari would be well aware by now that this is definitely within `<first>`'s capability.